src/analyzer.py

```diff
@@ -138,3 +140,38 @@ def noun_phrase(input_text):
     for chunk in doc.noun_chunks:
         n_phrase_lst.append(str(chunk))
     return n_phrase_lst
+
+
+def senti(tokens_column, sign):
+    """Create columns for positive and negative words"""
+    # Start off with an empty list
+    display_series = []
+    # For each item in the already existing tokens column in the data frame,
+    for token_element in tokens_column:
+        words = create_word_list(token_element)
+        # Add the words at the end of the list to the display series since
+        # they have the most positive sentiment value
+        if sign == 1:
+            display_series.append(", ".join(words[::-1][0:3]))
+        else:
+            display_series.append(", ".join(words[0:3]))
+    # Return an entire series based on the display_series list
+    return pd.Series(display_series)
+
+
+def create_word_list(token_element):
```

Comment:
Same with the naming here. Maybe `sorted_sentiment_word_list` or something like that, so it's clearer that the function is not just creating _any_ list, but a list specifically for the sentiment analysis, with all the words sorted based on sentiment.